src/cdk/_index.scss

```diff
@@ -1,4 +1,5 @@
-@forward './overlay/overlay' show overlay;
+@forward './overlay/overlay' show overlay, $z-index-overlay-container, $z-index-overlay,
+  $z-index-overlay-backdrop, $dark-backdrop-background;
 @forward './a11y/a11y' show a11y-visually-hidden, high-contrast;
 @forward './text-field/text-field' show text-field-autosize, text-field-autofill,
-  text-field-autofill-color;
+  text-field-autofill-color, text-field;
```

Comment:
I had to export the deprecated `text-field` mixin in order for the migration to work.

Reply:
Could you add a comment that explains why this is necessary and that we'd want to remove it eventually?